templates/ApiGear/Source/ApiGear/Private/unrealolink.cpp

```diff
@@ -1,7 +1,7 @@
 #include "unrealolink.h"
 #include "ApiGearSettings.h"
 THIRD_PARTY_INCLUDES_START
-#include "olink/clientregistry.h"
```

Comment:
shouldn't the path prefix stay, it tells that it is not "local" file but from some other part of code 
or is it now harder with this (nicely!) extracted module?

Reply:
I was thinking about the same, but thought I remove it first and see how it goes.